environment.yml

```diff
@@ -37,8 +38,7 @@ dependencies:
   - wheel
   - xz
   - zlib
-  - pip
   - pip:
     - eventio~=1.0
-
-
+    - zstandard
```

Comment:
What is this addition for?

Reply:
To read simtel files compressed using the zstd compression

Reply:
This was used already for prod4 and will i think be used for prod5 as well. Its much faster than gzip at a similar compression rate.

Reply:
Isn't this already required by pyeventio then?

Reply:
It's an optional dependency of pyeventio, so we need to install ot explicitly

Reply:
Otherwise you will get a `You need zstandard to read this file` error